tests/system/requests/conftest.py

```diff
@@ -19,6 +19,10 @@
 
 from tests.system import utils
 
+import http
+
+http.client.HTTPConnection.debuglevel = 5
```

Comment:
Should this be here?

Reply:
Yes this was needed here for the tests to run, not sure why the dependency there occurs though - looking into it now.